app/Commands/SyncCommand.php

```
@@ -0,0 +1,349 @@
+
+            $result = json_decode((string) $response->getBody(), true);
+
+            if (is_array($result)) {
+                $sent = count($payload);
+                $created = $result['created'] ?? 0;
+                $updated = $result['updated'] ?? 0;
+                $failed = $result['failed'] ?? 0;
+            }
+        } catch (GuzzleException $e) {
+            $this->error('Failed to push to cloud: '.$e->getMessage());
+            $failed = count($payload ?? []);
```

PHPStan (2.1.33)

Variable $payload on left side of ?? always exists and is not nullable.


(nullCoalesce.variable)